source/libsass.html.haml

```diff
@@ -51,7 +51,7 @@ title: libSass
   %li#javascript
     :markdown
       ### JavaScript
-      The only pure JavaScript implementation is [sass.js](https://github.com/medialize/sass.js). There's a [browser-based way to test it out](http://medialize.github.io/sass.js/), too.
+      The only pure JavaScript implementation is [sass.js](https://github.com/medialize/sass.js). There's a [browser-based way to test it out](http://medialize.github.io/playground.sass.js/), too.
```

Comment:
Yep. It's actually in a markdown filter. Just a line break will do. Make sure it's at the same level of indention though. Thanks!